docs/components/waitlist/overview.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title: '`<Waitlist />` component'
+description: The <Waitlist /> component renders a waitlist form that allows users to join for early access to your application.
+---
+
+![The \<Waitlist /> component renders a waitlist form that allows users to join for early access to your app.](/docs/images/ui-components/waitlist.svg)
+
+The `<Waitlist />` component renders a waitlist form that allows users to join for early access to your app.
```

Comment:
❓ Do we need to repeat the waitlist term here?

```suggestion
![The \<Waitlist /> component renders a form that allows users to join for early access to your app.](/docs/images/ui-components/waitlist.svg)

The `<Waitlist />` component renders a form that allows users to join for early access to your app.
```

Reply:
@kostaspt the first is the image alt text 🙃